core/timeMeasure.go

```diff
@@ -0,0 +1,112 @@
+package core
+
+import (
+	"fmt"
+	"sync"
+	"time"
+)
+
+type timeMeasure struct {
```

Comment:
maybe rename to `stopWatch` ? and the `Finish` method to `Stop` ?

Reply:
done